### PR TITLE
healthcare API: fix path for FhirResourceGetHistory and make output d…

### PR DESCRIPTION
**Pull request by noerog**
…escription clearer

Fixes b/156008376

### CODE REVIEW

File: healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java [hunk 1]
```diff
@@ -37,15 +37,15 @@
...
+  private static final String FHIR_NAME = "projects/%s/locations/%s/datasets/%s/fhirStores/%s/fhir/%s/%s";
   private static final JsonFactory JSON_FACTORY = new JacksonFactory();
   private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
 
   public static void fhirResourceGetHistory(String resourceName, String versionId)
       throws IOException, URISyntaxException {
     // String resourceName =
     //    String.format(
-    //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id");
+    //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id", "resource-type", "resource-id");
```
Comment (lesv) [inline healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java:48]: There are 6 `%s` in `FHIR_NAME` and 7 params here.  Something is amis.

Comment (dzlier-gcp) [reply healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java:48]: Ah looks like "store-id" and "fhir-id" were meant to be "FHIR store ID" and "FHIR Resource ID". You can delete "fhir-id" or "store-id" or combine them into "fhir-store-id".

File: healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public static void fhirResourceGetHistory(String resourceName, String versionId)
       responseEntity.writeTo(System.err);
       throw new RuntimeException();
     }
-    System.out.println("FHIR resource history list retrieved: ");
+    System.out.println("FHIR resource retrieved from version: ");
```
Comment (dzlier-gcp) [inline healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java:77]: Does the test check for this text in the output?

File: healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java (hunk 1 above)
Comment (noerog) [reply healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java]: Whoops, thanks. Deleted "fhir-id".

File: healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java (hunk 2 above)
Comment (noerog) [reply healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java:77]: Just fixed the test, thanks.

**Pull request merged**